sway/sway.5.txt

```diff
@@ -361,7 +361,8 @@ The default colors are:
 
 **output** <name> <background|bg> <file> <mode>::
 	Sets the wallpaper for the given output to the specified file, using the given
-	scaling mode (one of "stretch", "fill", "fit", "center", "tile").
+	scaling mode (one of "stretch", "fill", "fit", "center", "tile"). This image
+	must be in PNG format unless sway was built with gdk_pixbuf.
```

Comment:
Then how are future occurrences of #895 meant to be avoided? Trying to load a JPEG with the PNG loader gives the rather unhelpful "out of memory" error message.